static/app/views/settings/account/apiNewToken.tsx

```diff
@@ -46,31 +61,37 @@ export default class ApiNewToken extends Component {
             )}
           </TextBlock>
           <Panel>
-            <PanelHeader>{t('Create New User Auth Token')}</PanelHeader>
+            <PanelHeader>{t('Permissions')}</PanelHeader>
             <ApiForm
               apiMethod="POST"
               apiEndpoint="/api-tokens/"
-              initialData={{scopes: SORTED_DEFAULT_API_ACCESS_SCOPES}}
+              initialData={{scopes: []}}
               onSubmitSuccess={this.onSubmitSuccess}
               onCancel={this.onCancel}
               footerStyle={{
                 marginTop: 0,
                 paddingRight: 20,
               }}
+              submitDisabled={Object.values(this.state.permissions).every(
+                value => value === 'no-access'
+              )}
               submitLabel={t('Create Token')}
             >
               <PanelBody>
-                <FormField name="scopes" label={t('Scopes')} inline={false} required>
-                  {({name, value, onChange}) => (
-                    <MultipleCheckbox onChange={onChange} value={value} name={name}>
-                      {API_ACCESS_SCOPES.map(scope => (
-                        <MultipleCheckbox.Item value={scope} key={scope}>
-                          {scope}
-                        </MultipleCheckbox.Item>
-                      ))}
-                    </MultipleCheckbox>
-                  )}
-                </FormField>
+                <PermissionSelection
+                  appPublished={false}
+                  permissions={{
```

Comment:
should this be `this.state.permissions`? otherwise maybe pull this out into a const since you initialize the state to this too

Reply:
yes it'd be a lot better to use `this.state.permissions`, switching to that